src/app/backend/resource/pod/common.go

```diff
@@ -46,7 +46,7 @@ func getPodStatus(pod v1.Pod, warnings []common.Event) PodStatus {
 	}
 }
 
-// getPodStatus returns one of three pod statuses (pending, success, failed)
+// getPodStatus returns one of four pod statuses (pending, running, success, failed)
```

Comment:
It's still wrong.

Reply:
Should it be this:
(pending, running, succeeded, failed)

Also @maciaszczykm is there a method to propogate
v1.PodUnknown ?

Reply:
s/getPodStatus/getPodStatusPhase